reactive/cuda.sh

```diff
@@ -166,7 +164,7 @@ function all::all::add_cuda_path() {
 /usr/local/cuda/lib64
 EOF
 
-    cat > /etc/ld.so.conf.d/nvidia.conf << EOF
+    [ -d "/usr/local/nvidia" ] && cat > /etc/ld.so.conf.d/nvidia.conf << EOF
```

Comment:
this path didn't exist on my gpu-enabled instances, so i added this test before updating ld.so.conf.d and profile.d.